Result/ResultProtocol.swift

```diff
@@ -120,14 +114,13 @@ public func &&& <L: ResultProtocol, R: ResultProtocol> (left: L, right: @autoclo
 	return left.flatMap { left in right().map { right in (left, right) } }
 }
 
-infix operator >>- {
-	// Left-associativity so that chaining works like you’d expect, and for consistency with Haskell, Runes, swiftz, etc.
-	associativity left
-
-	// Higher precedence than function application, but lower than function composition.
-	precedence 100
+precedencegroup ChainingPrecedence {
+	associativity: left
+	higherThan: TernaryPrecedence
```

Comment:
I wasn't at all sure about the name or the higher/lower-than, but tried to pick what I thought made sense based on the docs.